src/components/datepicker/demoValidations/index.html

```diff
@@ -30,10 +30,10 @@ <h4>Inside a md-input-container</h4>
           md-max-date="ctrl.maxDate"></md-datepicker>
 
         <div ng-messages="myForm.dateField.$error">
-          <div ng-message="valid">The entered value is not a date!</div>
-          <div ng-message="required">This date is required!</div>
-          <div ng-message="mindate">Date is too early!</div>
-          <div ng-message="maxdate">Date is too late!</div>
+          <div ng-message="valid">Use a valid date format</div>
+          <div ng-message="required">This date is required</div>
```

Comment:
Let's get rid of the `required` message since we don't have `required` on the `md-datepicker`. That doesn't make for a good example for others to follow :)